docs/DeveloperGuide.md

```diff
@@ -398,7 +434,26 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+2. _{ more test cases …​ }_
+
+### Editing an employee
+
+1. Editing an employee while list of employees are shown
+
+    1. Prerequisites: List all persons using the `list` command. Multiple employees in the list.
+
+    2. Test case: `edit 1 n/Tsang Wei Ooi`<br>
+        Expected: First employee's name is updated with `Tsang Wei Ooi`. Details of the edited employee is shown in status message. Timestamp in the status bar is updated.
+
+    3. Test case: `edit`<br>
+        Expected: No data is being edited. Error details shown in status message. Status bar remains the same.
+
+    4. Test case: `edit 1 p/91234567 s/2000`<br>
+        Expected: First employee's phone number and salary is updated as `91234567` and `2000` respectively together. Details of edited contact shown in the status message. Timestamp in the status bar is updated.
+
+    5. Other incorrect edit commands to try: `edit x` (where x is larger than list size), `edit 1 s/-200` (salary should not be negative) etc.
+        Expected: No employee data is being edited. Error details shown in the status message. Status bar remains the same.
```

Comment:
Maybe to standardise the formatting with other clauses, you can consider adding a "<br>" before "Expected:" to start a new line?